CONTRIBUTING.md

```diff
@@ -149,7 +149,7 @@ There are two main pieces of the Operator `API` and `Controller`. Depending on w
     
 - Controllers:
 
-    We rely on (Ginkgo)[https://github.com/onsi/ginkgo] testing framework in our controllers E2E tests. If your made controller-related changes(/controllers/**), please make sure you have updated the tests and always run them.
+    We rely on [Ginkgo](https://github.com/onsi/ginkgo) testing framework in our controllers E2E tests. If your made controller-related changes(/controllers/**), please make sure you have updated the tests and always run them.
```

Comment:
Not related to the changes in this PR, but I don't want to open a new one just to make this small change.